app/src/main/java/com/framgia/wsm/screen/requestoff/RequestOffViewModel.java

```diff
@@ -31,4 +48,80 @@ public void onStart() {
     public void onStop() {
         mPresenter.onStop();
     }
+
+    @Bindable
+    public boolean isVisibleLayoutCompanyPay() {
+        return mIsVisibleLayoutCompanyPay;
+    }
+
+    public void setVisibleLayoutCompanyPay(boolean visibleLayoutCompanyPay) {
+        mIsVisibleLayoutCompanyPay = visibleLayoutCompanyPay;
+        notifyPropertyChanged(BR.visibleLayoutCompanyPay);
+    }
+
+    @Bindable
+    public boolean isVisibleLayoutInsuranceCoverage() {
+        return mIsVisibleLayoutInsuranceCoverage;
+    }
+
+    public void setVisibleLayoutInsuranceCoverage(boolean visibleLayoutInsuranceCoverage) {
+        mIsVisibleLayoutInsuranceCoverage = visibleLayoutInsuranceCoverage;
+        notifyPropertyChanged(BR.visibleLayoutInsuranceCoverage);
+    }
+
+    public void onPickTypeRequestOff(View view) {
+        mDialogManager.dialogListSingleChoice(mContext.getString(R.string.off_type),
+                R.array.off_type, mCurrentPositionOffType,
+                new MaterialDialog.ListCallbackSingleChoice() {
+                    @Override
+                    public boolean onSelection(MaterialDialog materialDialog, View view,
+                            int positionType, CharSequence charSequence) {
+                        setCurrentPositionOffType(positionType);
+                        setCurrentOffType(String.valueOf(charSequence));
+                        setLayoutOffType(positionType);
+                        return true;
+                    }
+                });
+    }
+
+    private void setLayoutOffType(int positionType) {
+        if (positionType == PositionOffType.OFF_HAVE_SALARY_COMPANY_PAY) {
+            setVisibleLayoutCompanyPay(true);
+        } else {
+            setVisibleLayoutCompanyPay(false);
+        }
+        if (positionType == PositionOffType.OFF_HAVE_SALARY_INSURANCE_COVERAGE) {
+            setVisibleLayoutInsuranceCoverage(true);
+        } else {
+            setVisibleLayoutInsuranceCoverage(false);
+        }
+        if (positionType == PositionOffType.OFF_NO_SALARY) {
+            setVisibleLayoutCompanyPay(false);
+            setVisibleLayoutInsuranceCoverage(false);
+        }
+    }
+
+    @Bindable
+    public String getCurrentOffType() {
+        return mCurrentOffType;
+    }
+
+    public void setCurrentOffType(String currentOffType) {
+        mCurrentOffType = currentOffType;
+        notifyPropertyChanged(BR.currentOffType);
+    }
+
+    public void setCurrentPositionOffType(int currentPositionOffType) {
+        mCurrentPositionOffType = currentPositionOffType;
+    }
+
+    @IntDef({
+            PositionOffType.OFF_HAVE_SALARY_COMPANY_PAY,
+            PositionOffType.OFF_HAVE_SALARY_INSURANCE_COVERAGE, PositionOffType.OFF_NO_SALARY
+    })
+    @interface PositionOffType {
+        int OFF_HAVE_SALARY_COMPANY_PAY = 0;
+        int OFF_HAVE_SALARY_INSURANCE_COVERAGE = 1;
```

Comment:
ERROR: 'member def type' have incorrect indentation level 8, expected level should be 4.